e2e/runner_test.go

```diff
@@ -26,60 +28,38 @@ func testRunner(t *testing.T) {
 		require.NoError(t, err)
 		acknowledgement.WaitForStreamToBeReady(stream)
 
-		resp, err := client.RunnerClient.Create(context.Background(), &pb.CreateRunnerRequest{
-			ServiceHash: testServiceHash,
-			Env:         []string{"BAR=3", "REQUIRED=4"},
-		})
+		msg, err := runBuilder.Start(engineAddress, testServiceStruct, []string{"BAR=3", "REQUIRED=4"})
 		require.NoError(t, err)
-		testRunnerHash = resp.Hash
+		testRunnerHash = lcdBroadcastMsg(msg)
 
 		// wait for service to be ready
 		_, err = stream.Recv()
 		require.NoError(t, err)
 	})
 
 	t.Run("recreate", func(t *testing.T) {
-		_, err := client.RunnerClient.Delete(context.Background(), &pb.DeleteRunnerRequest{Hash: testRunnerHash})
+		msgDelete, err := runBuilder.Stop(engineAddress, testRunnerHash, testServiceStruct, true)
```

Comment:
same here separating the stoping function to the actual message

Reply:
done